sickrage/providers/GenericProvider.py

```diff
@@ -122,10 +124,36 @@ def download_result(self, result):
         return False
 
     def find_propers(self, search_date=None):
-        results = self.cache.listPropers(search_date)
+        results = []
+        db = DBConnection()
+        placeholder = ','.join([str(x) for x in Quality.DOWNLOADED + Quality.SNATCHED + Quality.SNATCHED_BEST])
+        sql_results = db.select(
+            'SELECT s.show_name, e.showid, e.season, e.episode, e.status, e.airdate'
+            ' FROM tv_episodes AS e'
+            ' INNER JOIN tv_shows AS s ON (e.showid = s.indexer_id)'
+            ' WHERE e.airdate >= ' + str(search_date.toordinal()) +
+            ' AND e.status IN (' + placeholder + ')'
+        )
+
+        for sql_result in sql_results:
+            show_obj = Show.find(sickbeard.showList, int(sql_result[1])) if sql_result[1] else None
+
```

Comment:
I had to change this to number because I got this error:

```
    show_obj = Show.find(sickbeard.showList, int(sql_result[u'showid'])) if sql_result[u'showid'] else None
IndexError: Index must be int or string

I found this:
"pysqlite does not accept a Unicode string in row['string'] and will raise "IndexError: Index must be int or string"

but why it works in TorrentProvider.py and NZBProvider.py and not in GenericProvider.py ?